examples/ai_society/role_playing_multiprocess.py

```diff
@@ -181,6 +183,42 @@ def main() -> None:
     # Disable/Enable Printing
     verbose = True
 
+    # Check for tasks folder and install if not exists
+    # Define the folder path
+    folder_path = "./ai_society_data/tasks/"
+
+    # Check if the folder already exists
+    if not os.path.exists(folder_path):
+        os.makedirs(folder_path)
+
+    # Check if the folder is empty
+    if not os.listdir(folder_path):
+        # Download the zip file from the Google Drive link
+        file_id = "194PPaSTBR07m-PzjS-Ty6KlPLdFIPQDd"
+        url = f"https://drive.google.com/uc?id={file_id}"
+        response = requests.get(url)
+
+        # Save the zip file
+        zip_file_path = os.path.join(folder_path, "tasks.zip")
+        with open(zip_file_path, "wb") as f:
+            f.write(response.content)
+
+        # Extract the files from the zip file
+        with zipfile.ZipFile(zip_file_path, "r") as zip_ref:
+            zip_ref.extractall(folder_path)
+
+        # Delete the zip file
+        os.remove(zip_file_path)
+
+    # Remove any nested folder created during extraction
+    nested_folder_path = os.path.join(folder_path, "tasks")
+    if os.path.exists(nested_folder_path):
+        for file_name in os.listdir(nested_folder_path):
+            file_path = os.path.join(nested_folder_path, file_name)
+            destination_path = os.path.join(folder_path, file_name)
+            os.rename(file_path, destination_path)
+        os.rmdir(nested_folder_path)
```

Comment:
I don't think we need to have two snippets of codes that do the same thing. Can we add it as a function to `utils`?

Reply:
Sure! On it.

Reply:
@lightaime please take a look now this must solve the issue